src/Data/Functor/Contravariant/Divisible.hs

```diff
@@ -73,48 +78,54 @@ import GHC.Generics
 --
 -- A 'Divisible' contravariant functor is the contravariant analogue of 'Applicative'.
 --
--- In denser jargon, a 'Divisible' contravariant functor is a monoid object in the category
--- of presheaves from Hask to Hask, equipped with Day convolution mapping the Cartesian
--- product of the source to the Cartesian product of the target.
---
--- By way of contrast, an 'Applicative' functor can be viewed as a monoid object in the
--- category of copresheaves from Hask to Hask, equipped with Day convolution mapping the
--- Cartesian product of the source to the Cartesian product of the target.
+-- Continuing the intuition that 'Contravariant' functors consume input, a 'Divisible'
+-- contravariant functor also has the ability to composed "beside" another contravariant
+-- functor.
 --
--- Given the canonical diagonal morphism:
+-- Serializers provide a good example of 'Divisible' contravariant functors. To begin
+-- let's start with the type of serializers for specific types:
 --
 -- @
--- delta a = (a,a)
+-- newtype Serializer a = Serializer { runSerializer :: a -> ByteString }
 -- @
 --
--- @'divide' 'delta'@ should be associative with 'conquer' as a unit
+-- This is a contravariant functor:
 --
 -- @
--- 'divide' 'delta' m 'conquer' = m
--- 'divide' 'delta' 'conquer' m = m
--- 'divide' 'delta' ('divide' 'delta' m n) o = 'divide' 'delta' m ('divide' 'delta' n o)
+-- instance Contravariant Serializer where
+--   contramap f s = Serializer . runSerializer s . f
 -- @
 --
--- With more general arguments you'll need to reassociate and project using the monoidal
--- structure of the source category. (Here fst and snd are used in lieu of the more restricted
--- lambda and rho, but this construction works with just a monoidal category.)
+-- That is, given a serializer for @a@ (@s :: Serializer a@), and a way to turn
+-- @b@s into @a@s (a mapping @f :: b -> a@), we have a serializer for @b@:
```

Comment:
This sentence is what made it click for me, the only difficulty I had when reading this was that `a` and `b` are reversed compared to the type signature of contramap.

Reply:
Specifically, this is what made `Contravariant` click for me. 👍